internal/server/semantic_cache.proto

```
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package ember.v1;
```

Buf (1.66.1)

[error] Files with package "ember.v1" must be within a directory "ember/v1" relative to root but were in directory "internal/server".

(PACKAGE_DIRECTORY_MATCH)